website/source/docs/providers/aws/r/iam_access_key.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "aws"
+page_title: "AWS: aws_iam_access_key"
+sidebar_current: "docs-aws-resource-iam-access-key"
+description: |-
+  Provides an IAM access key. This is a set of credentials that allow API requests to be made as an IAM user.
+---
+
+# aws\_iam\_access\_key
+
+Provides an IAM access key. This is a set of credentials that allow API requests to be made as an IAM user.
+
+## Example Usage
+
+```
+resource "aws_iam_user" "lb" {
+    name = "loadbalancer"
+    path = "/system/"
+}
+
+resource "aws_iam_access_key" "lb" {
+    user = "${aws_iam_user.lb.name}"
+    status = "Active"
+}
+
+resource "aws_iam_user_policy" "lb_ro" {
+    name = "test"
+    user = "${aws_iam_user.lb.name}"
+    policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "ec2:Describe*"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+EOF
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `user` - (Required) The IAM user to associate with this access key.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The access key ID.
```

Comment:
One quick question @bitglue -- I don't see `id` in the `Schema:` declaration for `resource_aws_iam_access_key`, but I'm assuming this is discoverable through a different means. Is that the case? Just want to make sure I've got the docs correct for the other resources.